vispy/app/backends/_wx.py

```diff
@@ -218,13 +218,18 @@ def __init__(self, *args, **kwargs):
         else:
             self._gl_context = p.context.shared.ref._gl_context
 
+        if p.position == None:
+            pos = wx.DefaultPosition
+        else:
+            pos = p.position
+
         if p.parent is None:
             style = (wx.MINIMIZE_BOX | wx.MAXIMIZE_BOX | wx.CLOSE_BOX |
                      wx.SYSTEM_MENU | wx.CAPTION | wx.CLIP_CHILDREN)
             style |= wx.NO_BORDER if not p.decorate else wx.RESIZE_BORDER
             style |= wx.STAY_ON_TOP if p.always_on_top else 0
-            self._frame = wx.Frame(None, wx.ID_ANY, p.title, p.position,
-                                   p.size, style)
+            self._frame = wx.Frame(None, wx.ID_ANY, p.title, pos, p.size, style)
```

Comment:
Flake is also complaining about this line being one character too long. Please move one or more of the parameters to the next line.

Reply:
ok